weather.js

```diff
@@ -25,11 +39,11 @@ async function getWeather(input) {
  */
 async function getCoordinates(input) {
     const geoURL = `${apiURLPrefix}geo/1.0/direct?q=${input},us&appid=${process.env.W_API}`;
-    let data = await buildResponseBody(geoURL);
-    if (data.length === 0) {
-        return console.error(`${input} is not a valid location, please try again.`);
+    try {
+        return await buildResponseBody(geoURL);
```

Comment:
This feels weird.

When I see 'buildResponseBody' I think that the request was already made.
Having that method DO the request feels wrong.
I would do the request in this method and then hand off the response object to `buildResponseBody `and rename it something like `parseResponse` or something.